kombu/asynchronous/timer.py

```diff
@@ -11,25 +11,26 @@
 
 from vine.utils import wraps
 
-from kombu.log import get_logger
+from kombu.log import get_logg
```

Comment:
🤔  Is it a typo?

Reply:
🤦 yes

Reply:
surprised that our build is working despite this